release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -1139,6 +1139,10 @@ Kubernetes 1.29 removed the following deprecated APIs, so you must migrate manif
 [id="ocp-4-16-bare-metal-hardware-bug-fixes_{context}"]
 ==== Bare Metal Hardware Provisioning
 
+* Previously, newer versions of Redfish used Manager resources to deprecate the Uniform Resource Identifier (URI) for the RedFish Virtual Media API. This caused any hardware that used the newer Redfish URI for Virtual Media to not be provisioned. With this release, the Ironic API identifies the correct Redfish URI to deploy for the RedFish Virtual Media API so that hardware relying on either the deprectaed or newer URI could be provisioned. (link: https://issues.redhat.com/browse/OCPBUGS-30171[*OCPBUGS-30171*]
```

Comment:
```suggestion
* Previously, newer versions of Redfish used Manager resources to deprecate the Uniform Resource Identifier (URI) for the RedFish Virtual Media API. This caused any hardware that used the newer Redfish URI for Virtual Media to not be provisioned. With this release, the Ironic API identifies the correct Redfish URI to deploy for the RedFish Virtual Media API so that hardware relying on either the deprectaed or newer URI could be provisioned. (link:https://issues.redhat.com/browse/OCPBUGS-30171[*OCPBUGS-30171*])
```

Reply:
:heavy_check_mark: